odd-platform-ui/src/components/DataEntityDetails/TestReport/TestReportItem/TestReportItem.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { DataQualityTest } from 'generated-sources';
+import { Collapse, Grid, Typography } from '@material-ui/core';
+import TestReportTypeItem, {
+  DataSetTestReportTypeNames,
+} from 'components/shared/TestReportTypeItem/TestReportTypeItem';
+import cx from 'classnames';
+import MinusIcon from 'components/shared/Icons/MinusIcon';
+import PlusIcon from 'components/shared/Icons/PlusIcon';
+import TestItem from 'components/DataEntityDetails/TestReport/TestReportItem/TestItem/TestItem';
+import { dataEntityTestPath } from 'lib/paths';
+import { Link } from 'react-router-dom';
+import { StylesType } from './TestReportItemStyles';
+
+interface TestReport {
```

Comment:
You have very similar interface in another component. Please consider moving it to redux/interfaces and reusing where necessary.